test/expected/bgw_db_scheduler.out

```diff
@@ -78,6 +78,8 @@ CREATE TABLE public.bgw_log(
     application_name TEXT,
     msg TEXT
 );
+CREATE VIEW sorted_bgw_log AS
+    SELECT * FROM bgw_log ORDER BY mock_time, msg_no, application_name;
```

Comment:
Please you update the PR to order by `mock_time, application_name, msg_no`. `msg_no` ordering only makes sense within an application / process, so if we order first by `application_name`, then we can verify that within a specific process, messages appear in the correct order. For the same `mock_time`, cross-application log messages are unfortunately concurrent, so there is no fixed ordering (which is in fact why the nondeterminism existed in the first place).